cmd/libs/go2idl/client-gen/generators/generator_for_clientset.go

```diff
@@ -142,7 +143,10 @@ func (c *Clientset) Discovery() $.DiscoveryInterface|raw$ {
 var newClientsetForConfigTemplate = `
 // NewForConfig creates a new Clientset for the given config.
 func NewForConfig(c *$.Config|raw$) (*Clientset, error) {
-	var clientset Clientset
+  if c.RateLimiter == nil && c.QPS > 0 {
+    c.RateLimiter = flowcontrol.NewTokenBucketRateLimiter(c.QPS, c.Burst)
+  }
+  var clientset Clientset
```

Comment:
It seems we are using "\t" in all other places in this file. Can you please change it to it here too?